src/mds/Mantle.h

```diff
@@ -0,0 +1,34 @@
+#include <lua.hpp>
+#include <list>
+#include <map>
+using std::list;
```

Comment:
Please avoid `using` directives in headers (we do already have some but I'd rather not add more)